CONTRIBUTING.md

```diff
@@ -1,6 +1,11 @@
 # Como contribuir
+
+[English Version]()
```

Comment:
you forgot the url in this link, but I think it isn't necessary because has more bellow other link.

Reply:
I have removed this unused link